source/common/quic/envoy_quic_server_session.cc

```diff
@@ -146,6 +148,30 @@ void EnvoyQuicServerSession::OnRstStream(const quic::QuicRstStreamFrame& frame)
                                                    /*from_self*/ false, /*is_upstream*/ false);
 }
 
+void EnvoyQuicServerSession::setHttp3Options(
```

Comment:
Why does the server session not need something like `disable_keepalive_` from the client session?